core/dns.go

```diff
@@ -116,52 +80,67 @@ func (dnsResolver *DNSResolver) LookupTXT(hostname string) ([]string, time.Durat
 	return txt, rtt, err
 }
 
-// LookupHost uses a DNSSEC-enabled query to find all A/AAAA records associated with
-// the provided hostname. If the query fails due to DNSSEC, error will be
-// set to ErrorDNSSEC.
+// LookupHost sends a DNS query to find all A/AAAA records associated with
+// the provided hostname.
 func (dnsResolver *DNSResolver) LookupHost(hostname string) ([]net.IP, time.Duration, error) {
 	var addrs []net.IP
 	var answers []dns.RR
 
 	m := new(dns.Msg)
 	m.SetQuestion(dns.Fqdn(hostname), dns.TypeA)
-	r, aRtt, err := dnsResolver.LookupDNSSEC(m)
+
+	r, aRtt, err := dnsResolver.ExchangeOne(m)
```

Comment:
While we're refactoring, could we just move this part inside of ExchangeOne()?

> m := new(dns.Msg)
> m.SetQuestion(dns.Fqdn(hostname), dns.TypeA)

Then just have the interface be `ExchangeOne(hostname, type)`.